bertopic/_bertopic.py

```diff
@@ -3663,6 +3663,32 @@ def _combine_zeroshot_topics(self,
         self.__dict__.clear()
         self.__dict__.update(merged_model.__dict__)
```

Comment:
I spoke too soon! `merged_model.__dict__` replaces `self.outliers` with 0, so the outliers are never repositioned a couple of lines down.

Reply:
Maybe it's just best to set `outliers = self._outliers` just before merging on line 3627, then using this on line 3668 instead?

Will also have to set `self._outliers = 1` inside the if statement

Reply:
I tested this out and it looks good to me

<img width="643" alt="outliers" src="https://github.com/MaartenGr/BERTopic/assets/58509816/86fcf5f0-e27f-4564-9886-9aafe4880ba3">

Reply:
Awesome, thanks for checking this out! I just pushed a commit which should finally resolve this issue. If you could do a final check, I can merge this PR.